### PR TITLE
Testing test

### PR DESCRIPTION
**Pull request by sr-meh**
- Implements tests for the event model.
- Update the README

### CODE REVIEW

File: config/schedule.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+#
+# every 4.days do
+#   runner "AnotherModel.prune_old_records"
+# end
+
+# Learn more: http://github.com/javan/whenever
+
+#every 1.day, at: '9:00 am' do
+  #runner 'Event.send_reminder_emails'
+#end
```
Comment (juperameza) [inline config/schedule.rb]: If  you dont need this delete it 
```suggestion

```

**Review by juperameza — CHANGES_REQUESTED** [review]
Everything looks good just that nick pick of not leaving comments o your code. Also update the branch :+1: great job guys

**Review by PudinDeveloper — APPROVED** [review]
Same comments as @juperameza, everything else looks good. Well done.

**Review by hugocruz1234 — APPROVED** [review]

**Comment by Antonio3veces3** [comment]
Excellent!!!

**Pull request closed without merging**